lib/formatter.js

```diff
@@ -53,6 +51,10 @@ module.exports = function defaultFormatter(options, params, payload){
         return match;
     });
 
+    // replace bar
+    const bar = formatBar(params.progress, Object.assign({}, options, { restWidth: s.length - 5 })); // 5 = {bar}
```

Comment:
why is it moved outside the standard placeholder processing ?

Reply:
We need to know the length of the remaining string befor the bar could be computed.